ToFeet/tofeet
# feat/add_analysis_route

HOW TO TEST:

- SIGNUP: POST /api/signup BODY:
{
	"firstname": "",
	"lastname": "",
	"email": "",
	"password": ""
}
- LOGIN: POST /api/login BODY:
{
  "email": "",
  "password": ""
}

- SIMULATE DATA: 
GET /api/simulation?timestamp=XXX
GET /api/simulation/ill?timestamp=XXX

- START FIRST ANALYSIS: POST /api/analysis (Should return code: 2 -> model created)

- RE-SIMULATE DATA

- START ANALYSIS: POST /api/analysis (Should return code: 0 (No anomaly) or code: 1 (Anomaly detected)
